### ISSUE TITLE
😄2019 Roadmap & RFC 😄

### ISSUE THREAD

**Issue by iamclaytonray** [konpa/devicon#148]
Hello community! 😄😄😄

I was added as a collaborator earlier today and I'm in the process of figuring out next steps for this library. Because we are coming up on the end of the year, please do not expect hardcore work to be done on this project until 2019. Most issues and PRs will not be resolved until after the new year due to the holiday season and my other non-OSS obligations. With that being said, I'd like to formalize some processes to make sure we are on a good track for 2019 and to ensure everyone is happy with the outcome.

Clayton

# UPDATE 2020

After becoming a maintainer, I got a swamped with work. I'm sorry guys!

**Comment by Berkmann18** [comment]
Not sure if you already set that up or have examples in sight but for PR/Issues templates, but you could have a look at [this repo](https://github.com/stevemao/github-issue-templates) which I've based myself on to set up a [template repo](https://github.com/Berkmann18/TemplateJS) (which contains settings for bots that helps with a variety of things, code analysers like BCH/Codacy and of course a fairly complete set of [labels](https://github.com/Berkmann18/TemplateJS/labels) and [PR/issue templates](https://github.com/Berkmann18/TemplateJS/tree/master/.github)).

**Comment by iamclaytonray** [comment]
This is great! I'll definitely draw from here and make tweaks as needed!

I started a project that I abandoned some time ago but would love to pick it up again, especially if you'd be down to collab on it! It's similar to what you are doing (though, I didn't get to that level of completion). Essentially, it's a convention or "spec" on github flows, like issues, pull request templates, issue templates, labels, etc.
